spec/expectations_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'spec_helper'
+
+describe RubyExpectationsHook do
+  def req(expectations, content)
+    struct expectations: expectations, content: content
+  end
+
+  def compile_and_run(request)
+    runner.run!(runner.compile(request))
+  end
+
+  let(:runner) { RubyExpectationsHook.new(mulang_path: './bin/mulang') }
+  let(:result) { compile_and_run(req(expectations, code)) }
+
+  context 'smells' do
+    let(:code) { 'x = 4' }
+    let(:expectations) { [] }
+
+    it { expect(result).to eq [{binding: 'x', inspection: 'HasTooShortBindings'}] }
+  end
+
+
+  context 'expectations' do
+    let(:code) { 'pepita = Object.new' }
```

Comment:
The test is ok, but Ruby's content doesn't use this form. A more realistic example could be defining a `module Pepita` and then checking for `DeclaresObject:Pepita`.